terraform/interpolate.go

```diff
@@ -52,12 +54,38 @@ func (i *Interpolater) Values(
 			mod = i.Module.Child(scope.Path[1:])
 		}
 		for _, v := range mod.Config().Variables {
-			for k, val := range v.DefaultsMap() {
-				result[k] = ast.Variable{
-					Value: val,
+			name := fmt.Sprintf("var.%s", v.Name)
```

Comment:
This is eventually destined to change when we do maps in the same way as lists - but for now `DefaultsMap` is inlined, yes.